app/views/toggle/index.html.erb

```diff
@@ -2,110 +2,137 @@
 <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 
 <style>
-
-  .row-container {
-    display: flex;
-    justify-content: space-between;
-    padding: 20px;
-    border-bottom: solid 1px #efe7e7;
-  }
-
-  .deactivated-row {
-    color: #c7c7c7;
-  }
-
-  .row-container h3 {
-    margin: 0px;
-  }
-
-  .ident {
-    color: #ababab;
-  }
-
-  body {
-    font-family: 'Roboto', sans-serif;
-    max-width: 700px;
-    margin-left: auto;
-    margin-right: auto;
-    background-color: #e8e8e8;
-  }
-
-  /* The switch - the box around the slider */
-  .switch {
-    position: relative;
-    display: inline-block;
-    width: 60px;
-    height: 34px;
-  }
-
-  /* Hide default HTML checkbox */
-  .switch input {
-    opacity: 0;
-    width: 0;
-    height: 0;
-  }
-
-  /* The slider */
-  .slider {
-    position: absolute;
-    cursor: pointer;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    background-color: #ccc;
-    -webkit-transition: .4s;
-    transition: .4s;
-  }
-
-  .slider:before {
-    position: absolute;
-    content: "";
-    height: 26px;
-    width: 26px;
-    left: 4px;
-    bottom: 4px;
-    background-color: white;
-    -webkit-transition: .4s;
-    transition: .4s;
-  }
-
-  input:checked + .slider {
-    background-color: #2196F3;
-  }
-
-  input:focus + .slider {
-    box-shadow: 0 0 1px #2196F3;
-  }
-
-  input:checked + .slider:before {
-    -webkit-transform: translateX(26px);
-    -ms-transform: translateX(26px);
-    transform: translateX(26px);
-  }
-
-  /* Rounded sliders */
-  .slider.round {
-    border-radius: 34px;
-  }
-
-  .slider.round:before {
-    border-radius: 50%;
-  }
-
-  .card {
-    background-color: white;
-    padding: 20px;
-
-    box-shadow: 1px 1px 13px 0px #d8d5d5;
-    border-radius: 6px;
-  }
-    .section {
-      padding: 20px;
-      text-align: center;
-      background-color: #f7f7f7;
-      font-size: 21px;
-    }
+.toggle-table {
```

Comment:
CSS changes. Most of these not changed, just line inserted some lines ontop of it